tests/test_interface.sh

```diff
@@ -38,12 +38,23 @@ else
   fail "-e without folder did not fail as expected"
 fi
 
-#test 3: running with an invalid flag (-z)
+#Test 3: running with an invalid flag (-z)
 output=$(TESTING_SKIP_ROOT_CHECK=1 "$CC_SNAPSHOT" -z 2>&1) && status=0 || status=$?
 if [[ $status -ne 0 && "$output" == *"usage:"* ]]; then
   pass "Invalid flag (-z) is handled with error"
 else
   fail "Invalid flag (-z) did not trigger error as expected"
 fi
 
+#Test 4: Dry-run does not error and prints each step
+if output=$(sudo "$CC_SNAPSHOT" -d mytest 2>&1); then
+  if [[ "$output" == *"[DRY_RUN]"* ]]; then
```

Comment:
I wonder if we should parse a few more things from the output. We may want to make sure we have N number of [DRY_RUN] messages printed, just in case it prints one as a fluke but then isn't working as expected.